docassemble/AssemblyLine/data/questions/al_visual.yml

```diff
@@ -311,7 +317,7 @@ template: al_share_answers_message_template
 subject: |
   ${ AL_ORGANIZATION_TITLE } form from ${ al_share_form_from_name }
 content: |
-  ${ share_interview_answers_message }
+  ${ share_interview_answers_message.replace("\n", "<br>") }
```

Comment:
Are we always sending HTML email?

Reply:
Yeah, docassemble always sends an HTML version. Relevant code is: https://github.com/jhpyle/docassemble/blob/70d7dbfd7db5075f1e51d2b64906739a8f86f679/docassemble_base/docassemble/base/util.py#L8366-L8371

Specifically, when you pass in a template, it gets the content of that template in HTML. BeautifulSoup strips out the tags for the plaintext version if the client doesn't support HTML (which most email clients support, and default to).